src/providers/splash.ts

```diff
@@ -0,0 +1,53 @@
+import { AmountWithDecimal, Provider } from "@/types/core";
+import { AmmPool, makeNativePools, Pools } from "@ergolabs/ergo-dex-sdk";
+import { AssetAmount, Explorer, RustModule } from "@ergolabs/ergo-sdk";
+
+class Splash implements Provider {
+  private baseUrl: string;
+
+  constructor(url: string) {
+    this.baseUrl = url;
+  }
+
+  async getOrderBook(marketId: string) {
+    try {
+      const response = await fetch(
```

Comment:
The type of this variable is implicitly `any`, which is not a good thing. You should explicitly define the type for this variable based on the structure of the response.

Reply:
I fixed it.